other-page/index.html

```diff
@@ -9,19 +9,21 @@
     <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="../styles/reset.css">
     <link rel="stylesheet" href="../styles/style.css">
-    <link rel="stylesheet" href="../styles/home.css">
+    <link rel="stylesheet" href="other-page.css">
     <title>Document</title>
 </head>
 
 <body>
     <header>
         <div class="gutter-right logo">
-            <img src="../assets/big-logo.png" />
-        </div>
+            <a href="../other-page" >See All Concerts</a>
```

Comment:
maybe consider renaming other-page to something like concerts or festivals? If you do, be sure you're going through all the pages that reference it so that it doesn't break the site.